tests/src/common/WskCli.java

```diff
@@ -353,7 +352,7 @@ public String createAction(int expectedCode, String name, String artifact, Strin
         }
 
         if (shared) {
-            cmd = Util.concat(cmd, new String[] { "--shared" });
+            cmd = Util.concat(cmd, new String[] { "--shared", "yes"});
```

Comment:
The Cobra argument processing package we are using in Go does not support the option to enabled the --shared flag to have optional arguments. We can either have it so --shared with no arguments enables publishing, or we can have the --shared yes/no option. We can't have both.